pallets/nfts/src/default_weights.rs

```diff
@@ -1,10 +1,16 @@
 use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
 
 impl crate::WeightInfo for () {
-    fn create() -> Weight {
-        (31_410_000 as Weight)
-            .saturating_add(DbWeight::get().reads(2 as Weight))
-            .saturating_add(DbWeight::get().writes(3 as Weight))
+    fn create(series_id: u32) -> Weight {
+        if series_id == 0 {
+            (24_111_000 as Weight)
+                .saturating_add(DbWeight::get().reads(1 as Weight))
+                .saturating_add(DbWeight::get().writes(2 as Weight))
+        } else {
+            (31_410_000 as Weight)
+                .saturating_add(DbWeight::get().reads(2 as Weight))
+                .saturating_add(DbWeight::get().writes(3 as Weight))
+        }
```

Comment:
You probably do not want to modify the generated files, instead you may prefer to use something closer to what parity does: https://github.com/paritytech/substrate/blob/bc43ad44618ba8757fc280c0bb79ddce9228f62f/frame/collective/src/lib.rs#L316.

I think you could use the same syntax with an if statement inside

Reply:
It should be better now :)